compass/ocean/tests/tides/dem/__init__.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+import compass.ocean.tests.tides.dem.dem_pixel as dem_pixel
+from compass.step import Step
+
+
+class CreatePixelFile(Step):
+    """
+    A step for creating a pixel file for creating MPAS meshes
+
+    Attributes
+    ----------
+    """
+
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.tides.init.Init
+            The test case this step belongs to
+        """
+        super().__init__(test_case=test_case, name='pixel',
+                         ntasks=1, min_tasks=1, openmp_threads=1)
+
+        self.add_input_file(
```

Comment:
Building these DEM datasets could be done once and cached somewhere, as it's pretty expensive and is typically only redone when GEBCO release a new bathymetry (once per year).